src/Strings.cpp

```diff
@@ -0,0 +1,158 @@
+#include "Strings.h"
+#include <random>
+#include <time.h>
+
+
+Strings::Strings() {
+    std::srand(time(0));
+
+    strings.insert({ "Title", "Match Made in Hell" });
+    strings.insert({ "CurrentLevel", "The Current Level is: " });
+    strings.insert({ "MenuText", "Welcome to Match Made in Hell!\nPress s to go to the settings menu\nPress enter to start" });
+    strings.insert({ "Settings", "Settings Menu\n" });
+    strings.insert({ "SettingsText", "Press Backspace to return to the main menu\nPress the key you wish to change and then press\nthe key you want." });
+    strings.insert({ "CurrentKeys", "The current keybindings are : \n" });
+    strings.insert({ "MoveRight", "Move Right(RightArrow):" });
+    strings.insert({ "MoveLeft", "Move Left(LeftArrow): " });
+    strings.insert({ "Jump", "Jump(UpArrow): " });
+    strings.insert({ "Shoot", "Shoot(Spacebar): " });
+    strings.insert({ "NewLine", "\n" });
+    strings.insert({ "ChangeLevel", "\nTo load progress press R" });
```

Comment:
Change this key to "LoadLevel". Also, let's change the key they click to "L" for Load